src/renderer/components/attachment/menuAttachment.tsx

```diff
@@ -127,7 +127,7 @@ const MenuAttachment = ({
         }
         position={Position.TOP_LEFT}
       >
-        <Button minimal icon='paperclip' />
+        <Button id='test-attachment-menu' minimal icon='paperclip' />
```

Comment:
?

Reply:
without an id the test script can not easily find it

Reply:
```suggestion
        <Button id='attachment-menu' minimal icon='paperclip' />
```

Reply:
the idea of the `test-` prefix was to mark it as only used in tests, some other id's are used also by desktop